apps/condo/domains/common/components/Pickers/DateRangePicker.tsx

```diff
@@ -20,6 +23,29 @@ const RANGE_PICKER_CSS = css`
 `
 
 const DateRangePicker: React.FC<RangePickerSharedProps<Dayjs>> = (props) => {
+    const { onChange, ...restProps } = props
+
+    const { logEvent, getEventName } = useTracking()
+
+    const eventName = getEventName(TrackingEventType.Daterange)
+
+    const onChangeCallback: RangePickerSharedProps<Dayjs>['onChange'] = (values, formatString) => {
+        if (eventName && formatString.length) {
+            const eventProperties = { component: { value: formatString } }
+
+            const componentId = get(restProps, 'id')
```

Comment:
it's better to use `props.id`
```suggestion
            const componentId = get(props, 'id')
```